src/ucm.ejs

```diff
@@ -4,6 +4,18 @@
   <head>
     <meta charset="utf-8">
     <title>Unison Codebase</title>
+    <% if (webpackConfig.mode === "production") {%>
+      <script type="text/javascript">
+        const pathSegments = location.pathname.split("/").filter(p => p !== "");
+
+        if (pathSegments.length >= 2) {
+          const basePath = pathSegments.slice(0, 2).join("/")
+          const newBase = document.createElement("base");
+          newBase.setAttribute("href", "/" + basePath + "/")
+          document.getElementsByTagName("head")[0].appendChild(newBase);
+        }
+      </script>
+    <% } %>
```

Comment:
If we're in production mode generate a `<base>` element with the first 2 parts of the URL and insert it into the head of the document (`<base href="/:token/ui/">`).